### PR TITLE
Confirmation before rest

### PR DESCRIPTION
**Pull request by tamiror6**
detailed in issue #76

### CODE REVIEW

File: popup/popup.js [hunk 1]
```diff
@@ -51,12 +51,15 @@ async function listenForClicks() {
 }
 
 async function handleReset() {
-    const tabs = await browser.tabs.query({
-        currentWindow: true,
-        active: true
-    }).catch(onError);
-    sendMessageToTabs(tabs, {"reset": "true"});
-    alert("Please refresh your browser tab to apply.");
+    const confirm = window.confirm("Are you sure? reset is an irreversable action")
```
Comment (norbit8) [inline popup/popup.js]: *irreversible
```suggestion
    const confirm = window.confirm("Are you sure? reset is an irreversible action")

```

File: package-lock.json [hunk 2]
```diff
@@ -1,6476 +1,8 @@
 {
   "name": "moodlebooster",
   "version": "1.0.0",
-  "lockfileVersion": 2,
+  "lockfileVersion": 1,
   "requires": true,
-  "packages": {
```
Comment (norbit8) [inline package-lock.json]: Please don't touch this file.

**Review by norbit8 — CHANGES_REQUESTED** [review]

**Comment by norbit8** [comment]
p.s. you don't have to close the PR like you did [here](https://github.com/norbit8/MoodleBooster/pull/83) just push a fix from the same branch.

**Comment by tamiror6** [comment]
> p.s. you don't have to close the PR like you did [here](https://github.com/norbit8/MoodleBooster/pull/83) just push a fix from the same branch.

it was branch from fork by mistake

File: package-lock.json (hunk 2 above)
Comment (tamiror6) [reply package-lock.json]: When a contributor do npm install it changes automatically changes package.lock.
usually there is git.ignore file will solve this problem, I will open issue for this

**Pull request closed without merging**